spec/spec_helper.rb

```diff
@@ -52,11 +52,14 @@ module PuppetSpec
   require behaviour.relative_path_from(Pathname.new(dir))
 end
 
+require 'webmock/rspec'
 require 'vcr'
 VCR.configure do |vcr|
   vcr.cassette_library_dir = File.expand_path('vcr/cassettes', PuppetSpec::FIXTURE_DIR)
   vcr.hook_into :webmock
   vcr.configure_rspec_metadata!
+  # Uncomment to debug vcr
+  # vcr.debug_logger = $stderr
```

Comment:
Should this stay? If so, as a nit can we update :61 to be "Uncomment next line to debug vcr" just so we're being specific?